compose/project.py

```diff
@@ -638,7 +638,10 @@ def _find():
         if remove_orphans:
             for ctnr in orphans:
                 log.info('Removing orphan container "{0}"'.format(ctnr.name))
-                ctnr.kill()
+                try:
+                    ctnr.kill()
+                except Exception:
```

Comment:
We should catch `docker.errors.APIError` here as to not hide other legitimate exceptions.